fairlearn/datasets/_fetch_credit_card.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Fairlearn Contributors.
```

Comment:
This is incomplete. We still need to update `fairlearn/datasets/__init__.py` and `test/unit/datasets/test_datasets.py`.

Reply:
Done.